src/pages/model/[model]/color/[color].astro

```diff
@@ -20,8 +20,8 @@ export async function getStaticPaths() {
       DEVICE_MANAGER.getDeviceListByModel(_modelId);
     allModelDevices.forEach((d: Device) => {
       const modelColorInfo = {
-        params: { model: _modelId, color: d.color.id },
-        props: { targetModel: _modelId, targetColor: d.color.id },
+        params: { model: _modelId, color: d.color?.id ?? "default" },
+        props: { targetModel: _modelId, targetColor: d.color?.id ?? "default" },
```

Comment:
Sorry, don't get this, when we will don't have color id?

The list is control by use, and in this PR diff, I see all device have color.

Reply:
sor missed this, please see below comment

Reply:
I know. But we do provide the key for all device. The value and format are control by us. We can make the data as `"color": "default"` to make the code simpler?

Reply:
So you are suggesting changing

```
--- "color": null
+++ "color": "default"
```

So that when usage it will changes from 

```
--- d.color?.id ?? "default"
+++ d.color === "default" ? "default" : d.color.id
```

Do you mean above? If so IMO it does not `make the code simpler` by much

But to put this discussion further, maybe we should change the entire `device_info.json` so that it would become `model.deviceColorVariations`, instead of one-color-one-device

Reply:
Something like this:

```
"color": {
         "id": "default",
         "name": "default",
         "hexcode": "#default???"
},
```